docs/tutorial.rst

```diff
@@ -178,6 +204,39 @@ If you run ``nox --list-sessions``, you'll see that this generates the following
 
 If you only want to run one of the parametrized sessions, see :ref:`running_paramed_sessions`.
 
+Giving friendly names to parametrized sessions
+----------------------------------------------
+
+The automatically generated names for parametrized sessions, such as ``tests(django='1.9', database='postgres')``, can be long and unwieldy to work with even with using :ref:`keyword filtering <opt-sessions-and-keywords>`. You can give parametrized sessions custom IDs to help in this scenario. These two examples are equivalent:
```

Comment:
👎 on long lines, they make diffs harder to parse (for future updates). Of course re-flowing text also is bad for diffs so pick your poison, but I personally prefer a set limit like 80, 100 or 120 chars.

Reply:
Actually limiting lines to x chars doesn't solve the problem of diffs. I prefer to follow this https://rhodesmill.org/brandon/2012/one-sentence-per-line/